### DISCUSSION TITLE
Add social media icon

### DISCUSSION THREAD

**Discussion by natalianunez** [adityatelange/hugo-PaperMod#429] [Q&A]
Hey all,

I'm trying to add social media icon but it don't render.

1. The config.yml
![image](https://user-images.githubusercontent.com/22640871/120254863-2a2adb00-c261-11eb-976a-34821e376b70.png)

2. The html generated
![image](https://user-images.githubusercontent.com/22640871/120254710-cef8e880-c260-11eb-98a4-3453f9ab6b01.png)

3. The page
![image](https://user-images.githubusercontent.com/22640871/120254984-60685a80-c261-11eb-946c-c486b0052525.png)

Thanks for any help!

**Comment by adityatelange** [comment]
I guess that is because of Firefox addon..
